packages/design-system/src/styleguide/colors.stories.mdx

```diff
@@ -142,3 +142,16 @@ import ColorCircles from './ColorCircles.vue';
 		}}
 	</Story>
 </Canvas>
+
+## Sticky
+
+<Canvas>
```

Comment:
I spoke with Max for this. Need clarification from you side. Since in the future, maybe we can implement other themes colors and stickies colors as well.